browser/quick_search_engines/android/BUILD.gn

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Import required GN build configuration files
```

Comment:
These comments (also for android_library, sources, deps, etc.. aren't really necessary. Comments should explain our code, not how gn works

Reply:
Removed comments here : https://github.com/brave/brave-core/pull/26478/commits/31aa2a9153b05ccaa43b7611c1f27648ef2b01c6